TRLevelControlTests/TR1/IOTests.cs

```diff
@@ -0,0 +1,240 @@
+
+namespace TRLevelControlTests.TR1;
+
+[TestClass]
+[TestCategory("OriginalIO")]
+public class IOTests : TestBase
+{
+    [TestMethod]
+    public void ReadWriteGym()
+    {
+        ReadWriteTR1Level(TRLevelNames.ASSAULT);
+    }
+
+    [TestMethod]
+    public void ReadWriteCaves()
+    {
+        ReadWriteTR1Level(TRLevelNames.CAVES);
+    }
+
+    [TestMethod]
+    public void ReadWriteVilcabamba()
+    {
+        ReadWriteTR1Level(TRLevelNames.VILCABAMBA);
+    }
+
+    [TestMethod]
+    public void ReadWriteValley()
+    {
+        ReadWriteTR1Level(TRLevelNames.VALLEY);
+    }
+
+    [TestMethod]
+    public void ReadWriteToQ()
+    {
+        ReadWriteTR1Level(TRLevelNames.QUALOPEC);
+    }
+
+    [TestMethod]
+    public void ReadWriteCut1()
+    {
+        ReadWriteTR1Level(TRLevelNames.QUALOPEC_CUT);
+    }
+
+    [TestMethod]
+    public void ReadWriteFolly()
+    {
+        ReadWriteTR1Level(TRLevelNames.FOLLY);
+    }
+
+    [TestMethod]
+    public void ReadWriteColosseum()
+    {
+        ReadWriteTR1Level(TRLevelNames.COLOSSEUM);
+    }
+
+    [TestMethod]
+    public void ReadWriteMidas()
+    {
+        ReadWriteTR1Level(TRLevelNames.MIDAS);
+    }
+
+    [TestMethod]
+    public void ReadWriteCistern()
+    {
+        ReadWriteTR1Level(TRLevelNames.CISTERN);
+    }
+
+    [TestMethod]
+    public void ReadWriteToT()
+    {
+        ReadWriteTR1Level(TRLevelNames.TIHOCAN);
+    }
+
+    [TestMethod]
+    public void ReadWriteCut2()
+    {
+        ReadWriteTR1Level(TRLevelNames.TIHOCAN_CUT);
+    }
+
+    [TestMethod]
+    public void ReadWriteKhamoon()
+    {
+        ReadWriteTR1Level(TRLevelNames.KHAMOON);
+    }
+
+    [TestMethod]
+    public void ReadWriteObelisk()
+    {
+        ReadWriteTR1Level(TRLevelNames.OBELISK);
+    }
+
+    [TestMethod]
+    public void ReadWriteSanctuary()
+    {
+        ReadWriteTR1Level(TRLevelNames.SANCTUARY);
+    }
+
+    [TestMethod]
+    public void ReadWriteMines()
+    {
+        ReadWriteTR1Level(TRLevelNames.MINES);
+    }
+
+    [TestMethod]
+    public void ReadWriteCut3()
+    {
+        ReadWriteTR1Level(TRLevelNames.MINES_CUT);
+    }
+
+    [TestMethod]
+    public void ReadWriteAtlantis()
+    {
+        ReadWriteTR1Level(TRLevelNames.ATLANTIS);
+    }
+
+    [TestMethod]
+    public void ReadWriteCut4()
+    {
+        ReadWriteTR1Level(TRLevelNames.ATLANTIS_CUT);
+    }
+
+    [TestMethod]
+    public void ReadWritePyramid()
+    {
+        ReadWriteTR1Level(TRLevelNames.PYRAMID);
+    }
+
+    [TestMethod]
+    public void ReadWriteEgypt()
+    {
+        ReadWriteTR1Level(TRLevelNames.EGYPT);
+    }
+
+    [TestMethod]
+    public void ReadWriteCat()
+    {
+        ReadWriteTR1Level(TRLevelNames.CAT);
+    }
+
+    [TestMethod]
+    public void ReadWriteHive()
+    {
+        ReadWriteTR1Level(TRLevelNames.HIVE);
+    }
+
+    [TestMethod]
+    public void ReadWriteStronghold()
+    {
+        ReadWriteTR1Level(TRLevelNames.STRONGHOLD);
+    }
```

Comment:
Could these benefit from test parameterization?

Reply:
Great suggestion, thanks. I've added this in.